planning/behavior_velocity_planner/autoware_behavior_velocity_blind_spot_module/src/util.cpp

```diff
@@ -681,4 +684,57 @@ std::optional<StopPoints> generate_stop_points(
     std::nullopt, stop_points_list.instant_stopline, stop_points_list.critical_stopline};
 }
 
+tl::expected<double, std::string> calc_ego_to_blind_spot_lanelet_lateral_gap(
+  const autoware_utils::LinearRing2d & ego_footprint,
+  const lanelet::ConstLanelets & last_lanelets_before_turning,
+  const autoware::experimental::lanelet2_utils::TurnDirection & turn_direction)
+{
+  if (ego_footprint.size() != 7) {
+    return tl::make_unexpected("Unexpected ego_footprint's size.");
+  }
+
+  // left front is 6, left back is 4, right_front is 1, right_back is 3
+  const auto front_idx = (turn_direction == TurnDirection::Left) ? 6 : 1;
```

Comment:
please use these constants

Reply:
done in https://github.com/autowarefoundation/autoware_universe/pull/11265/commits/a4023d8cad8bffc9bdc68c0f6120cbf176a0d3a7